advanced/Scripts/whitelist.sh

```diff
@@ -128,81 +98,13 @@ RemoveDomain(){
   	echo "::: $1 is NOT whitelisted! No need to remove"
   	fi
   else
-    #Domain is in the whitelist file, add to a temporary array and remove from whitelist file
-    #if $verbose; then
-    #echo "::: Un-whitelisting $dom..."
-    #fi
-    domToRemoveList=("${domToRemoveList[@]}" $1)
-    modifyHost=true
+    echo "$1" | sed 's/\./\\./g' | xargs -I {} perl -i -ne'print unless /'{}'(?!.)/;' ${whitelist}
   fi
```

Comment:
But if you could write that up and submit it as a PR instead of as line item comments. We can't pull in line comments, only commits and PR's. So feel free to add that to either a new branch and PR or as another push to your existing branch that you have PR'd. (If you have a PR open, I don't remember offhand.)